app/models/content_item.rb

```diff
@@ -15,7 +20,13 @@ def self.create_or_replace(base_path, attributes)
     item.assign_attributes(attributes)
 
     if item.upsert
-      item.register_routes(previous_item: previous_item)
+      begin
+        item.register_routes(previous_item: previous_item)
+      rescue GdsApi::BaseError
```

Comment:
I'd imagine we'd want to revert if any exception is thrown here? could be that we just re-raise it if it's not an instance of `GdsApi::BaseError`?